evap/results/exporters.py

```diff
@@ -285,8 +310,13 @@ def write_questionnaire(self, questionnaire, evaluations_with_results, contribut
 
     # pylint: disable=arguments-differ
     def export_impl(
-        self, semesters, selection_list, include_not_enough_voters=False, include_unpublished=False, contributor=None
-    ):
+        self,
+        semesters: QuerySet[Semester],
+        selection_list,
+        include_not_enough_voters: bool = False,
+        include_unpublished: bool = False,
+        contributor: None | UserProfile = None,
```

Comment:
Optional

Reply:
Agreed, also applies in other places

Reply:
https://peps.python.org/pep-0604/

Reply:
My point was to swap the order here, so that `None` comes last since that's the order we usually use. If there's a way for us to also add a linting check for this, you can also add that